lib/paper_trail/record_trail.rb

```diff
@@ -313,14 +323,17 @@ def update_columns(attributes)
       record_update_columns(changes)
     end
 
+    # Given `@record`, when building the query for the `versions` association,
+    # what `item_type` (if any) should we use in our query. Returning nil
+    # indicates that rails should do whatever it normally does.
     def versions_association_item_type
       type_column = @record.class.inheritance_column
-      type_name = (respond_to?(type_column) ? send(type_column) : nil) ||
+      item_type = (respond_to?(type_column) ? send(type_column) : nil) ||
         @record.class.name
-      if type_name == @record.class.base_class.name
-        ""
+      if item_type == @record.class.base_class.name
+        nil
```

Comment:
(In connection with the test for .blank? from 184)
Not sure how much performance loss might be experienced, but the blank was an attempt to avoid the unscope / scope when item_type referred to the base_class.  For many STI implementations perhaps it doesn't matter because the majority of things might be subclassed, but in some cases it could provide a benefit.  (Family / CelebrityFamily kind of examples.)